src/Storage/Storage.Management/Models/PSLocalUser.cs

```diff
@@ -0,0 +1,243 @@
+        }
+
+        public PermissionScope ParsePermissionScope()
+        {
+            PermissionScope scope = new PermissionScope();
+            scope.Permissions = this.Permissions;
+            scope.Service = this.Service;
+            scope.ResourceName = this.ResourceName;
+            return scope;
+        }
+
+        public static PSPermissionScope[] GetPSPermissionScopes(IList<PermissionScope> scopes)
+        {
+            if (scopes == null)
+            {
+                return null;
+            }
+            List<PSPermissionScope> psscopes = new List<PSPermissionScope>();
+            foreach (PermissionScope scope in scopes)
+            {
+                psscopes.Add(new PSPermissionScope(scope));
+            }
+            return psscopes.ToArray();
+        }
+
+        public static List<PermissionScope> ParsePermissionScopes(PSPermissionScope[] psscopes)
+        {
+            if (psscopes == null)
+            {
+                return null;
+            }
+            List<PermissionScope> scopes = new List<PermissionScope>();
+            foreach (PSPermissionScope psscope in psscopes)
+            {
+                scopes.Add(psscope.ParsePermissionScope());
+            }
+            return scopes;
+        }
+    }
+
+    public class PSLocalUserKeys
+    {
+        public PSLocalUserKeys() { }
+
+        public PSLocalUserKeys(LocalUserKeys keys)
+        {
+            if (keys == null)
+            {
+                return;
+            }
+
+            this.SharedKey = keys.SharedKey;
+            this.SshAuthorizedKeys = PSSshPublicKey.GetPSSshPublicKeys(keys.SshAuthorizedKeys);
+        }
+
+        public PSSshPublicKey[] SshAuthorizedKeys { get; set; }
+        public string SharedKey { get; set; }
+    }
+
+    public class PSLocalUserRegeneratePasswordResult
+    {
+        public PSLocalUserRegeneratePasswordResult() { }
+
+        public PSLocalUserRegeneratePasswordResult(LocalUserRegeneratePasswordResult properties)
+        {
+                //this.Sid = properties.Sid;
+                //this.SharedKey = properties.SharedKey;
+                this.SshPassword = properties.SshPassword;
+                //this.HomeDirectory = properties.HomeDirectory;
+                //this.HasSharedKey = properties.HasSharedKey;
+                //this.HasSshKey = properties.HasSshKey;
+                //this.HasSshPassword = properties.HasSshPassword;
+                //this.SshAuthorizedKeys = PSSshPublicKey.GetPSSshPublicKeys(properties.SshAuthorizedKeys);
+                //this.PermissionScopes = PSPermissionScope.GetPSPermissionScopes(properties.PermissionScopes);
+        }
+
+        //public PSPermissionScope[] PermissionScopes { get; set; }
+        //public string HomeDirectory { get; set; }
+        //public PSSshPublicKey[] SshAuthorizedKeys { get; set; }
+        //public string Sid { get; }
+        //public string SharedKey { get; set; }
+        public string SshPassword { get; }
```

Comment:
We recommend to use `SecureString` for passwords

Reply:
@isra-fel 
This feature is already released in Storage public preview module.
If we change the string to SecureString , it will be a breaking change to the existing feature user.
Although we don't block breaking from preview to GA, I still don't prefer to break the existing script of the public preview user.

So is it possible we keep the output as what it is now?